builder/ghc-for-component-wrapper.nix

```diff
@@ -15,14 +15,19 @@
 }:
 
 let
-  inherit (configFiles) ghcCommand ghcCommandCaps packageCfgDir;
+  inherit (configFiles) targetPrefix ghcCommand ghcCommandCaps packageCfgDir;
   libDir         = "$out/${configFiles.libDir}";
   docDir         = "$out/share/doc/ghc/html";
+  # For musl we can use haddock from the buildGHC
```

Comment:
Should add a comment here as well, why we don't want to use the one built with musl. E.g.

> Using the one from the buildGhc is more efficient here, as we do not need to rebuild haddock with the musl compiler.